src/CMakeLists.txt

```diff
@@ -27,37 +23,15 @@
 #   -DCMAKE_BUILD_TYPE=type
 # to the cmake call.
 
-# Version 2.8.3 introduces CMakeParseArguments.
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.16)
 
-# Select a default compiler because CMake does not respect the PATH environment variable for some generators,
-# e.g. used by existing compute servers in Basel.
-# See https://stackoverflow.com/a/45934279 and https://issues.fast-downward.org/issue1031 for details.
-find_program(CMAKE_C_COMPILER NAMES $ENV{CC} cc gcc PATHS ENV PATH NO_DEFAULT_PATH)
-find_program(CMAKE_CXX_COMPILER NAMES $ENV{CXX} c++ g++ PATHS ENV PATH NO_DEFAULT_PATH)
-
-# Path containing custom CMake modules
-set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake_modules)
-include(FastDownwardMacros)
-
-fast_downward_default_to_release_build()
-project(fast-downward)
-fast_downward_report_bitwidth()
-# Due to a bug in cmake, configuration types are only set up correctly on the second cmake run.
-# This means that cmake has to be called twice for multi-config generators like Visual Studio.
-fast_downward_set_configuration_types()
-fast_downward_add_profile_build()
-
-set(FAST_DOWNWARD_MAIN_CMAKELISTS_READ TRUE)
+project(fast-downward
+    DESCRIPTION "Fast Downward is a domain-independent classical planning system."
+    HOMEPAGE_URL https://www.fast-downward.org/
+    LANGUAGES CXX)
```

Comment:
Does it make sense to use this `LANGUAGES` feature here? We only have the Python translator, and setting the language to C++ might make sense only for the project defined in `search`.

Reply:
If setting `LANGUAGES` in `search` works, this would be better. I think previously, we had to set it here because of the way we set compiler flags and settings like build types. For some of these, CMake needs to know which compiler is used and it uses the `LANGUAGES` for that. But now that the overall project doesn't do much anymore, it is probably fine to move it.

Reply:
We removed the setting here.